### PR TITLE
Standards report progress

### PR DESCRIPTION
**Pull request by dmcavoy**
In the standards report print view, calculate the number of lessons completed based on actual student progress. 

Screen Shot 2020-03-05 at 12 20 46 PM

## Links

- [jira](https://codedotorg.atlassian.net/browse/LP-1189)

## Testing story

Updated tests for StandardsReport and added a test for the updated getNumberLessonsCompleted method.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by Erin007 — APPROVED** [review]

File: apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js [hunk 1]
```diff
@@ -115,7 +115,24 @@ export function fetchStudentLevelScores(scriptId, sectionId) {
...
+        .stages;
+
+    stages.forEach(stage => {
+      const lessonCompletionStatus = getLessonCompletionStatus(state, stage.id);
+      if (lessonCompletionStatus.completed) {
+        lessonsCompleted += 1;
+      }
+    });
+  }
+  return lessonsCompleted;
```
Comment (Erin007) [inline apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js:135]: I wonder if we can re-use this here: https://github.com/code-dot-org/code-dot-org/blob/09ccb837f05fe71956ba0fd2c19865a9ca8d9de2/apps/src/templates/sectionProgress/standards/StandardsProgressTable.jsx#L156?

Comment (dmcavoy) [reply apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js:135]: They are doing different things. This new one is counting the number of lessons completed in the course. The one in the table is number of lessons completed for a specific standard.

Comment (dmcavoy) [reply apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js:135]: I'll rename the one in the table to make it more clear

**Review by Erin007 — APPROVED** [review]
Good find! Explicitly naming stages as plugged helps.  Thanks for fixing.

**Pull request merged**